[2023-05-01T17:31:05.143+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-05-01T17:31:03.861953+00:00 [queued]>
[2023-05-01T17:31:05.154+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-05-01T17:31:03.861953+00:00 [queued]>
[2023-05-01T17:31:05.155+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-05-01T17:31:05.155+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-05-01T17:31:05.155+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-05-01T17:31:05.170+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-05-01 17:31:03.861953+00:00
[2023-05-01T17:31:05.191+0000] {standard_task_runner.py:55} INFO - Started process 17957 to run task
[2023-05-01T17:31:05.199+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2023-05-01T17:31:03.861953+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/processing_user.py', '--cfg-path', '/tmp/tmpdamsm16p']
[2023-05-01T17:31:05.201+0000] {standard_task_runner.py:83} INFO - Job 50: Subtask is_api_available
[2023-05-01T17:31:05.276+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2023-05-01T17:31:03.861953+00:00 [running]> on host 9bd2cedcb1a8
[2023-05-01T17:31:05.351+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-05-01T17:31:03.861953+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-01T17:31:03.861953+00:00
[2023-05-01T17:31:05.352+0000] {http.py:120} INFO - Poking: api/
[2023-05-01T17:31:05.390+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-05-01T17:31:05.409+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-05-01T17:31:06.061+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-05-01T17:31:06.077+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20230501T173103, start_date=20230501T173105, end_date=20230501T173106
[2023-05-01T17:31:06.098+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-05-01T17:31:06.122+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
